File: bubble_sort_middle/gimme.py
def gimme(arr):
    sorted = list(arr) # guarda a array em uma variavel
    for i in range(0, len(sorted)):  #loop que percorre o tamanho da array

        for j in range(i+1, len(sorted)): #loop que percorre uma posição a frente o tamanho da lista

            if sorted[i] > sorted[j]:   #condicional que verifica se o primeiro 
                                        #elemento do index é maior que o segundo elemento do index                 
                temp = sorted[i] #armazena o valor do maior
                sorted[i] = sorted[j] #injeta o valor do menor na posição do maior
                sorted[j] = temp #injeta o valor na segunda posição 
    mid = sorted[1] #
    for i in range(len(arr)):
        if arr[i] == mid:
            return i

File: bubble_sort_middle/gimme.js
function gimme(arr) {
    let sorted = arr.slice()
    for (let i = 0; i < sorted.length; i++) {
        for (let j = i + 1; j < sorted.length; j++) {
            if (sorted[i] > sorted[j]) {
                let temp = sorted[i]
                sorted[i] = sorted[j]
                sorted[j] = temp
            }
        }
    }

    let mid = sorted[1]

    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === mid) {
            return i
        }
    }
}